tests/test_database_update.py

```diff
@@ -50,6 +50,42 @@ def test_database_update_s3():
     assert hash.unordered(filtered_output_file_path) == 107700257
 
 
+@pytest.mark.integration
+def test_database_update_s3_prefix():
+    """
+    Tests custom database update for bowtie 2 using an S3 prefix and assumed primary name
+
+    NOTE: to test auto-generation of database_primary_name, this should be run right after
+    test_database_update_s3
```

Comment:
I don't love interdependent, but separate, integration tests as they can have unintended consequences with future changes.  Better way to for the dependency would be have 1 test that calls add then update. Could force that for now by creating a wrapping test but that still separates code in a way that may not be clear if only looking at the underlying tests.